### PR TITLE
qa/tests: reduced powercycle jobs amount

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -67,7 +67,7 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 05 04 * * 1,6 CEPH_BRANCH=master; MACHINE_NAME=smithi;    /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s ceph-ansible -k distro -e $CEPH_QA_EMAIL
 ###  The suite below must run on bare-metal because it's performance suite and run 3 times to produce more data points
 57 03 * * 2,5 CEPH_BRANCH=master; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s perf-basic -k distro -e $CEPH_QA_EMAIL -N 3
-09 03 * * 6   CEPH_BRANCH=master; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s powercycle -k distro -e $CEPH_QA_EMAIL
+09 03 * * 4,6   CEPH_BRANCH=master; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s powercycle --subset 1/4 -k distro -e $CEPH_QA_EMAIL
```
Comment (jdurgin) [inline qa/crontab/teuthology-cronjobs:70]: should cycle through subsets like we do for rados

Comment (yuriw) [reply qa/crontab/teuthology-cronjobs:70]: no, i think if we just run as is that'd be good
disagree ?

Comment (jdurgin) [reply qa/crontab/teuthology-cronjobs:70]: if we don't change the numerator, we run exactly the same jobs, and don't get all the combinations

Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: See https://github.com/ceph/ceph/pull/35077

**Pull request closed without merging**